src/bun.js/node/types.zig

```diff
@@ -979,8 +979,14 @@ pub const ArgumentsSlice = struct {
 pub fn fileDescriptorFromJS(ctx: JSC.C.JSContextRef, value: JSC.JSValue, exception: JSC.C.ExceptionRef) ?bun.FileDescriptor {
     return if (bun.FDImpl.fromJSValidated(value, ctx, exception) catch null) |fd|
         fd.encode()
-    else
-        null;
+    else if (value.isObject()) { // FileHandle
```

Comment:
sorry, this is going to cause a performance issue in node:fs and we cannot do it this way